.github/workflows/master.yaml

```diff
@@ -0,0 +1,21 @@
+name: master
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
I think it's better to use the `v2` release instead of the master branch. More info: https://github.com/actions/checkout#checkout-v2